acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -54,7 +54,14 @@
   if agent['platform'] =~ /windows/
     arch = agent[:ruby_arch] || 'x86'
     base_url = ENV['MSI_BASE_URL'] || "http://builds.puppetlabs.lan/puppet-agent/#{ENV['SHA']}/artifacts/windows"
-    filename = ENV['MSI_FILENAME'] || "puppet-agent-#{ENV['VERSION']}-#{arch}.msi"
+
+    filename = ENV['MSI_FILENAME']
+    if !filename
+      Dir.mktmpdir do |tmp_dir|
+        latest = fetch(base_url, "LATEST-#{ARCH}", tmp_dir)
```

Comment:
AFAIK, we don't have a `fetch_as_string` sort of method, which would be preferable to writing a temp file here.